### PR TITLE
Adds liver, stomach and special augments

### PR DESCRIPTION
**Pull request by Time-Green**
Ports https://github.com/tgstation/tgstation/pull/28913 by @Tacolizard and @Cyberboss
Ports https://github.com/tgstation/tgstation/pull/29276 by @Tacolizard
Ports https://github.com/tgstation/tgstation/pull/28977 by @dannno  
:cl:
adds: Adds liver, stomach, augment manipulator and cybernetic organs you can print from research. Dont get EMP'd with those inside you. Ported from tacolizard and Danno.
/:cl:
I also got pissed at augment code and changed it. Old augment code basicly changed your normal limbs icon and a few variables. Right now, it swaps the whole fucking limb.

### CODE REVIEW

**Comment by ghost** [comment]
if you have time and feel like it, could you move disgust to the stomach? Its like that in tg too.

besides that,  good change.

File: code/modules/surgery/organs/stomach.dm [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	slot = "stomach"
+	attack_verb = list("gored", "squished", "slapped", "digested")
+	desc = "Onaka ga suite imasu."
+
+/obj/item/organ/stomach/on_life()
+	var/mob/living/carbon/human/H = owner
+
+	if(istype(H))
+		H.dna.species.handle_digestion(H)
+		H.dna.species.handle_disgust()
```
Comment (Cruix) [inline code/modules/surgery/organs/stomach.dm]: You need to send H do avoid a runtime every tick.

File: code/game/objects/items/robot/robot_parts.dm [hunk 2]
```diff
@@ -1,70 +1,15 @@
...
-	body_zone = "head"
-	var/obj/item/device/assembly/flash/handheld/flash1 = null
-	var/obj/item/device/assembly/flash/handheld/flash2 = null
-
 /obj/item/robot_parts/robot_suit
 	name = "cyborg endoskeleton"
 	desc = "A complex metal backbone with standard limb sockets and pseudomuscle anchors."
 	attack_verb = list("borged", "stated the laws of", "robotized")
 	hitsound = 'sound/voice/liveagain.ogg'
 	icon_state = "robo_suit"
```
Comment (Cruix) [inline code/game/objects/items/robot/robot_parts.dm]: set the icon to 'icons/obj/robot_parts.dmi' so it is not invisible.

**Comment by Time-Green** [comment]
Fuck there's a conflict. I'll fix it sometime this year

**Comment by Time-Green** [comment]
Eh fuck it it's fine, let's just merge it anyway

**Comment by Sharkeybtm** [comment]
Can you make auto implanters researchable at high tiers? Also make them reloadable with a long timer.

**Comment by Ratvar** [comment]
Sharkey, there is entire race dedicated to autoimplantion, and autoimplantion is powercreep. Augments are meant to be upgrades that you can get only if someone is willing to give you them, leaving you vulnerable in process.

**Comment by Time-Green** [comment]
This also counts for my other PRs:
Please tell me when to fix the conflicts, I don't want to fix them right now and have them break again

**Comment by Koffheatalt** [comment]
@Time-Green fix plz

**Comment by Time-Green** [comment]
I’ve been waiting for someone with merge power to say that, so I dont end up fixing it just to have it be abondoned again

**Comment by Time-Green** [comment]
@AsV9 Fixed the conflicts on this one. It was already reviewed by cruix.
Could you perhaps uhh, do the thing? Before it starts conflicting again?
EDIT: The massive changes are from the shitty maps we dont use, so meh

**Comment by Cruix** [comment]
If I reviewed it and found bugs then it needs to be reviewed again.

**Comment by Time-Green** [comment]
Goddamit. Also hold on, now it apparently doesn’t compile anymore

**Comment by Amelia0010** [comment]
merge master in, travis was being a shit

**Comment by Time-Green** [comment]
Thanks ❤️

**Comment by Time-Green** [comment]
All ready to merge!

**Comment by Ratvar** [comment]
*whistle*

**Comment by Smudged** [comment]
Soon I hope

**Pull request closed without merging**